BioscoopSysteemWeb/BioscoopSysteemWeb/wwwroot/css/app.css

```diff
@@ -1,5 +1,41 @@
 @import url('open-iconic/font/css/open-iconic-bootstrap.min.css');
 
+.socialFooter {
+    width:100%;
+    height:200px;
+    background-color:aliceblue;
+    display:flex;
+    justify-content:flex-end;
+}
+.socialContainer {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-right: 150px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    width: 100px;
+    align-content: flex-end;
+    justify-content: flex-end;
+}
+
+.socialRow {
+    flex-basis:33.333%;
+    }
+
+.socialTitle {
+    margin-bottom: 2px;
+    justify-self: baseline;
+    font-size: 20px;
+    margin-right:6px;
+}
+}
+
+.socialImg {
+    width: 25px;
+    height: auto;
+}
+

```

Comment:
Tip: We hebben Bootstrap in ons project zitten. Hiermee is het niet nodig om eigen CSS te maken maar kan je classes gebruiken van Bootstrap. Zie: https://getbootstrap.com/docs/5.3/getting-started/introduction/